environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -8,6 +8,10 @@ parameters:
   ContrailRepo:
     type: string
     default: ''
+  ContrailRepoNew:
```

Comment:
Personally I dont like such kind of names... 
Is it possible to keep just ContrailRepo and overwrite it is value to old value  for old nodes via node-specific-hieradata.yaml  ?

Reply:
It'll probably not work out that way. Thing is that this variable must be consumed at firstboot. And node-specific-hieradata comes into picture at a later point in time

Reply:
I suggest to check it. because if it works then the changes with repos are not needed at all.